pages/models/regulations/governors/GovCT2/index.md

```diff
@@ -0,0 +1,566 @@
+    the same as $$\omega$$, which is also backed by \[3\].
+
+## Outputs
+
+| name           | type  | unit | modelica name | IEC name | description      |
+|----------------|-------|------|---------------|----------|------------------|
+| $$P_\mathrm{m}$$ | float | pu   | PMechPu       | Pm       | mechanical power |
+
+## Equations & algorithm  
+
+–
+
+## Initial equations / boundary conditions (optional)
+
+The initial values for the system’s states are calculated from the
+initial mechanical power $$P_\mathrm{m\,0}$$ and rotation speed
+$$\omega_\mathrm{0}$$.
+
+### Helper variables
+
+<span id="eq-initPmnoloss">$$
+P_\mathrm{m\,noloss\,0} = 
+\begin{cases}
+    P_\mathrm{m\,0} + \omega_\mathrm{0} \cdot D_\mathrm{m},& \text{if } D_\mathrm{m}> 0\\
+    P_\mathrm{m\,0},              & \text{otherwise}
+\end{cases}
+ \qquad(1)$$</span>
+
+<span id="eq-initCfe">$$
+C_\mathrm{fe\,0} = 
+\begin{cases}
+    W_\mathrm{fnl} + P_\mathrm{m\,noloss\,0} / K_\mathrm{turb},& \text{if } K_\mathrm{turb}>0\\
+    W_\mathrm{fnl},              & \text{otherwise}
+\end{cases}
+ \qquad(2)$$</span>
+
+<span id="eq-initValve">$$
+V_\mathrm{0} = 
+\begin{cases}
+    C_\mathrm{fe\,0} / \omega_\mathrm{0},& \text{if } W_\mathrm{fspd}\\
+    C_\mathrm{fe\,0},              & \text{otherwise}
+\end{cases}
+ \qquad(3)$$</span>
+
+<span id="eq-initTex">$$
+\vartheta_\mathrm{ex\,0} = 
+\begin{cases}
+    C_\mathrm{fe\,0} \cdot \omega_\mathrm{0}^{D_\mathrm{m}},& \text{if } D_\mathrm{m}<0\\
+    1,              & \text{otherwise}
+\end{cases}
+ \qquad(4)$$</span>
+
+<span id="eq-initFsrt">$$
+F_\mathrm{srt\,0} = (P_\mathrm{ldref}/K_\mathrm{turb} + W_\mathrm{fnl} - \vartheta_\mathrm{ex\,0}) \cdot K_\mathrm{P\,load} + x_\mathrm{I\,load}
+ \qquad(5)$$</span>
+
+### Initial states
+
+<span id="eq-initXiload">$$
+x_\mathrm{I\,load\,0} = 1
+ \qquad(6)$$</span>
+
+<span id="eq-initXTurbine">$$
+x_\mathrm{turb\,0} = P_\mathrm{m\,noloss\,0}
+ \qquad(7)$$</span>
+
+<span id="eq-initXValve">$$
+x_\mathrm{valve\,0} = V_\mathrm{0}
+ \qquad(8)$$</span>
+
+<span id="eq-initXFdl">$$
+x_\mathrm{fdl\,ratelimit\,0} = K_\mathrm{turb}(V_\mathrm{max}-W_\mathrm{fnl})
+ \qquad(9)$$</span>
+
+<span id="eq-initXIGov">$$
+x_\mathrm{I\,gov\,0} = V_\mathrm{0}
+ \qquad(10)$$</span>
+
+<span id="eq-initXMeasPe">$$
+x_\mathrm{meas\,Pe\,0} = P_\mathrm{m\,0}
+ \qquad(11)$$</span>
+
+<span id="eq-initFsrt">$$
+x_\mathrm{fsrt\,ratelim\,0} = F_\mathrm{srt\,0}
+ \qquad(12)$$</span>
+
+<span id="eq-Texm">$$
+x_\mathrm{meas\,\vartheta\,ex\,0} = \vartheta_\mathrm{ex\,0}
+ \qquad(13)$$</span>
+
+<span id="eq-initTex">$$
+x_\mathrm{\vartheta\,ex\,0} = \vartheta_\mathrm{ex\,0}
+ \qquad(14)$$</span>
+
+<span id="eq-initXIMw">$$
+x_\mathrm{I\,MW} = 0
+ \qquad(15)$$</span>
+
+<span id="eq-initLastValue">$$
+x_\mathrm{last\,value\,0} = V_\mathrm{0}
+ \qquad(16)$$</span>
+
+<span id="eq-initFsra">$$
+x_\mathrm{fsra\,0} = 0
+ \qquad(17)$$</span>
+
+## Assumptions in modelica implementation (Open source implementations)
```

Comment:
in the open source implementation, the expectation is a table such as:

| Software      | URL | Language | Open-Source License | Last consulted date | Comments |
| --------------| --- | --------- | ------------------- |------------------- | -------- |
| Software name | [Link](https://github.com/toto) | modelica | [MPL v2.0](https://www.mozilla.org/en-US/MPL/2.0/)  | XX/0X/20XX | Comments can contain implementations details such as validation means, implementations key choices, etc. |

in the comment part, you can put the key assumptions behind modelica and DigSilent implementations, but the whole paragraph as it is might be too much for colib. however, having the details of this implementation on the dynawo repository would be very useful.

Reply:
I updated the document accordingly